DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html

```diff
@@ -37,6 +37,10 @@
                     </div>
                 </div>
             </div>
+            <div class="row" ng-include="">
+
+            </div>
         </div>
     </section>
-</section>
\ No newline at end of file
+</section>
+qweryyu
```

Comment:
include will be related for activity ui, cleared.
Second one is my bad, missed that, cleared.